dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -48,4 +48,7 @@ scratchdir.name=Directory where generated class files are created
 scratchdir=When this attribute is set, the JSPs are compiled to the specified directory instead of the server workarea directory.
 
 useJDKCompiler.name=Compile JSPs with the Java SDK compiler
-useJDKCompiler=When this attribute is set, the JSPs are compiled by using the Java SDK compiler.
\ No newline at end of file
+useJDKCompiler=When this attribute is set, the JSPs are compiled by using the Java SDK compiler.
+
+prepareJSPThreadCount.name=The number of threads to use when translating and compiling JSPs on startup.
+prepareJSPThreadCount=When prepareJSPs is enabled this configuration specifies the number of threads used.
```

Comment:
Change,
"When prepareJSPs is enabled this configuration specifies the number of threads used."
to,
"When prepareJSPs is enabled, this configuration specifies the number of threads used."